code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -47,7 +47,7 @@
 	///only used by humans.
 	var/obj/item/clothing/ears = null
 
-	/// Carbon
+	/// Carbon, should only be accessed through has_dna()
 	var/datum/dna/dna = null
```

Comment:
is this a safe assertion to make/rely on?

Reply:
in retrospect looking through the code, no not really, but people really should be using it. in an ideal world this would be shifted all the way down to carbon but this fucking sucks

Reply:
ok so we really shouldn't then, can you fix whatever that means IDK why you are here I didn't read close enough

Reply:
also all `has_dna()` does is return the `dna` variable or `null`.